batcher-ui/src/extra_utils/utils.ts

```diff
@@ -44,3 +46,12 @@ export const getEmptyOrderBook = () => {
     asks: [],
   };
 };
+
+export const getNetworkType = () => {
```

Comment:
Didn't this already exist somewhere?

Reply:
I deleted the existing function and added this reused function here.